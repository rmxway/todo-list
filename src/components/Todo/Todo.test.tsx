import { render } from '@testing-library/react';
import { Todo } from './';

describe('Todo', () => {
	it('Render is fine', () => {
		const { container } = render(<Todo id={1} completed message="Issue #1" onChange={() => null} />);

		expect(container).toMatchSnapshot();
	});
});
