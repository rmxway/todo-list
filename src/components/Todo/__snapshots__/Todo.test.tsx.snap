// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo Render is fine 1`] = `
<div>
  <label
    class="sc-aYaIB RffHk"
    for="1"
  >
    <i
      class="icon-check"
    />
    <input
      checked=""
      data-testid="1"
      id="1"
      name="1"
      role="checkbox"
      type="checkbox"
    />
    <span>
      Issue #1
    </span>
  </label>
</div>
`;
